mteb/abstasks/AbsTaskRetrieval.py

```diff
@@ -376,44 +376,78 @@ def _add_main_score(self, scores: ScoresDict) -> None:
     def calculate_metadata_metrics(self) -> None:
         self.load_data()
 
+        all_details = {}
         for split in self.metadata_dict["eval_splits"]:
+            all_details[split] = {}
             if self.is_multilingual:
                 for lang in self.relevant_docs.keys():
-                    process_language(
+                    print(split, lang)
+                    split_details = process_language(
                         self.relevant_docs[lang][split],
                         self.queries[lang][split],
                         self.corpus[lang][split],
                         lang,
                     )
+                    all_details[split][lang] = split_details
             else:
-                process_language(
+                print(split)
```

Comment:
Was this intentional or for debugging purposes?

Reply:
Same as above

Reply:
For this, as the number of splits are usually 1 or 2, perhaps this can be removed.